docs/kcl/revolve.md

```diff
@@ -6,7 +6,7 @@ layout: manual
 
 Rotate a sketch around some provided axis, creating a solid from its extent.
 
-This, like extrude, is able to create a 3-dimensional solid from a 2-dimensional sketch. However, unlike extrude, this creates a solid by using the extent of the sketch as its revolved around an axis rather than using the extent of the sketch linearly translated through a third dimension.
+This, like extrude, is able to create a 3-dimensional solid from a 2-dimensional sketch. However, unlike extrude, this creates a solid by using the extent of the sketch as its revolved around an axis rather than using the extent of the sketch linearly translated through a third dimension. Revolve occurs around a local sketch axis rather than a global axis.
```

Comment:
we should expose is_axis_2d to the user, in like a optional param "global" and then they can override this local only thing see discussion here: https://github.com/KittyCAD/modeling-app/issues/3959

Reply:
okay nevermind kurt says dont pull that thru and has valid reasons in that issue, you could fix the plane stuff he mentioned if you want but i also dont mind doing a pr for that

Reply:
didn't know how dirty you wanted to get into the rust side